### ISSUE TITLE
WCAG 4.1.2: Ensures ARIA attributes are allowed for an element's role (#ctl00_PlacerHolderContentRow_PlaceHolderMain_ctl19__ControlWrapper_RichImageField)

### ISSUE THREAD

**Issue by PolPasop** [cor-web/cor-design-system#1]
#### Issue
`Ensures ARIA attributes are allowed for an element's role` ([aria-allowed-attr](https://accessibilityinsights.io/info-examples/web/aria-allowed-attr))
#### Target application
[Zero Pollution Stakeholder Platform](https://prod-portal.cor.europa.eu/en/engage/Pages/zero-pollution-stakeholder-platform.aspx)
#### Element path
#ctl00_PlacerHolderContentRow_PlaceHolderMain_ctl19__ControlWrapper_RichImageField
#### Snippet
`<div id="ctl00_PlacerHolderContentRow_PlaceHolderMain_ctl19__ControlWrapper_RichImageField" class="ms-rtestate-field" style="display:inline" aria-labelledby="ctl00_PlacerHolderContentRow_PlaceHolderMain_ctl19_label">`
#### How to fix
    Fix all of the following:
      ARIA attribute cannot be used, add a role attribute or use a different element: aria-labelledby
#### Environment
Chrome version 100.0.4855.0
This accessibility issue was found using Accessibility Insights for Web 2.29.0 (axe-core 4.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
